core/src/test/java/io/substrait/type/proto/ExtensionRoundtripTest.java

```diff
@@ -196,6 +197,19 @@ void hashJoin() {
     verifyRoundTrip(relWithoutKeys);
   }
 
+  @Test
+  void nested_loop_join() {
```

Comment:
minor: the Java expectation is `camelCase` for function names.

Reply:
Thank you! This is my python subconscious taking over.. haha